backend/loan/templates/loan/room.html

```diff
@@ -412,7 +412,7 @@
   const roomName = {{ hashed_loan_id }};
 
   /* Web socket for chatroom conversation */
-  const chatSocket = new WebSocket(`ws://localhost:8000/ws/loan/chatroom/${roomName}/`);
+  let chatSocket = new WebSocket(`ws://${window.location.host}/ws/loan/chatroom/${roomName}/`);
```

Comment:
I think  it is better to remain `WebSocket(`ws://${window.location.host}/ws/loan/chatroom/${roomName}/`);` as `WebSocket(`ws://localhost:8000/ws/loan/chatroom/${roomName}/`);`.